jobs/transforms/base_model_audits.py

```diff
@@ -251,6 +252,59 @@ def process(self, input_model):
             yield audit_errors.ModelDomainObjectValidateError(input_model, e)
 
 
+@audit_decorators.AuditsExisting(base_models.BaseCommitLogEntryModel)
```

Comment:
@vojtechjelinek should there be a decorator here considering the the _get_domain_class method is Not Implemented? Isn't it enough to have the decorators in the derived classes?

Reply:
No, it shouldn't be here.

Reply:
So @YBCS Please remove it.

Reply:
@rohitkatlaa @vojtechjelinek Done PTAL!